src/blocks/custom_dbus.rs

```diff
@@ -63,12 +63,6 @@ struct Block {
 
 #[dbus_interface(name = "rs.i3status.custom")]
 impl Block {
-    async fn set_icon(&mut self, icon: &str) -> fdo::Result<()> {
-        self.widget.set_icon(icon)?;
```

Comment:
I guess `custom_dbus` should also have a format option (like `custom`)?

Reply:
I've added it named as "format" as in other blocks.